ckan/plugins/toolkit.py

```diff
@@ -141,7 +141,7 @@ def _initialize(self):
 current request.
 
 '''
-        t['h'] = getattr(pylons.config['pylons.h'], 'no_magic', None)
+        t['h'] = pylons.config['pylons.h']
```

Comment:
can we just use helpers.helper_functions now?  The code as-is has import order issues (configuration must be run before a module can import ckan.toolkit.h) but referencing the helper_functions AttributeDict might be safer.